exercises/practice/gilded-rose/.docs/instructions.md

```diff
@@ -0,0 +1,38 @@
+# Instructions
+
+Welcome to the engineering team at the Gilded Rose! As you know, we're an inn that buys and sells only the finest goods. Our goods are constantly degrading in quality as they approach their sell-by date, so we have an inventory system that keeps track of these data.
```

Comment:
Exercism md files by and large use one sentence per line. Could you split this up?

Reply:
Sure, done.